docs/traceflow-guide.md

```diff
@@ -0,0 +1,103 @@
+# Traceflow User Guide
+
+Antrea supports using Traceflow for network diagnosis: it generates tracing requests for traffic going through
+Antrea-managed Pod network. Creating a new Traceflow CRD triggers the Traceflow module to inject packet into OVS,
+provide various observation points along the packet's path and populate these observations into the status field of
+the Traceflow CRD. Users can start a new trace simply from either Kubectl or Antrea-Octant-Plugin and view Traceflow
+result via CRD or UI graph. We will also provide a corresponding Antctl to start a new trace in the near future.
+
+## Table of Contents
+
+- [Prerequisites](#Prerequisites)
+- [Start a New Trace](#Start-a-New-Trace)
+  - [Using kubectl and YAML file](#using-kubectl-and-YAML-file)
+  - [Using Octant with antrea-octant-plugin](#Using-Octant-with-antrea-octant-plugin)
+- [View Traceflow Result and Graph](#View-Traceflow-Result-and-Graph)
+- [View Traceflow CRDs](#View-Traceflow-CRDs)
+
+## Prerequisites
+You need to switch on traceflow from featureGates defined in antrea.yml for both Controller and Agent.
+```yaml
+  antrea-controller.conf: |
+    featureGates:
+    # Enable traceflow which provides packet tracing feature to diagnose network issue.
+      Traceflow: true
+  antrea-agent.conf: |
+    featureGates:
+    # Enable traceflow which provides packet tracing feature to diagnose network issue.
+      Traceflow: true
+```
+For antrea-octant-plugin installation, please refer to [antrea-octant-installation](/docs/octant-plugin-installation.md).
+
+## Start a New Trace
+You can choose to use either Kubectl together with YAML file or Octant UI to start a new trace.
+If you use Kubectl to start a new trace, we now support traceflow from Pod to either Pod or IP and
+you can also mark the protocol(TCP/UDP/ICMP) and desired ports of the trace. 
```

Comment:
mark the protocol? sounds incorrect to me. @antoninbas any comment?

Reply:
Agreed.

I would recommend the following:

If you use Kubectl to start a new trace, you can provide the following information which will be used to build the trace packet:
 * source Pod
 * destination Pod or destination IP address
 * transport protocol (TCP/UDP/ICMP)
 * transport ports

If you use the UI to start a new trace, we currently only support Pods as the destination, but will soon support destination IPs and Service names.

Reply:
Thanks for the suggestions. Have updated accordingly.